pkg/gardenlet/controller/seed/seed/components.go

```diff
@@ -77,14 +78,15 @@ import (
 )
 
 type components struct {
-	machineCRD    component.DeployWaiter
-	extensionCRD  component.DeployWaiter
-	etcdCRD       component.DeployWaiter
-	istioCRD      component.DeployWaiter
-	vpaCRD        component.DeployWaiter
-	fluentCRD     component.DeployWaiter
-	prometheusCRD component.DeployWaiter
-	persesCRD     component.DeployWaiter
+	machineCRD       component.DeployWaiter
+	extensionCRD     component.DeployWaiter
+	etcdCRD          component.DeployWaiter
+	istioCRD         component.DeployWaiter
+	vpaCRD           component.DeployWaiter
+	fluentCRD        component.DeployWaiter
+	prometheusCRD    component.DeployWaiter
+	persesCRD        component.DeployWaiter
+	openTelemetryCRD component.Deployer
```

Comment:
This should not make a difference, because you explicitly call `component.OpWait...`, but let's keep it unified
```suggestion
	openTelemetryCRD component.DeployWaiter
```